build.gradle

```diff
@@ -104,8 +104,10 @@ configure(projectsToConfigure) { projectToConf ->
     }
 
     testConfig junitVersion
-    testConfig 'org.easymock:easymock:3.1'
-    testConfig 'org.powermock:powermock-easymock-release-full:1.5.1'
+    testConfig 'org.easymock:easymock:3.5.1'
+    testConfig 'org.powermock:powermock-core:2.0.0'
```

Comment:
To be honest, I am not really sure if all of these powemock imports are really needed. I just added the ones that looked like they would be required....